paper-server/src/main/java/org/bukkit/craftbukkit/CraftWorldBorder.java

```diff
@@ -54,7 +56,8 @@ public void setSize(double newSize, TimeUnit unit, long time) {
         Preconditions.checkArgument(newSize >= 1.0D && newSize <= this.getMaxSize(), "newSize must be between 1.0D and %s", this.getMaxSize());
 
         if (time > 0L) {
-            this.handle.lerpSizeBetween(this.handle.getSize(), newSize, unit.toMillis(time), this.getWorld().getGameTime());
+            long gameTime = (this.getWorld() != null) ? this.getWorld().getGameTime() : 0; // Virtual Borders don't have a World
```

Comment:
I know i named the parameter gameTime initially but that's not really accurate, can rename it to the new name or your own.